src/_pytest/assertion/rewrite.py

```diff
@@ -750,12 +750,12 @@ def visit_Assert(self, assert_):
         the expression is false.
 
         """
-        if isinstance(assert_.test, ast.Tuple):
+        if isinstance(assert_.test, ast.Tuple) and len(assert_.test.elts) == 2:
```

Comment:
(probably got lost in the review noise, but this should be >= 1 to catch the following)

```python
assert(
    'very long thing but has a trailing comma',
)
```

Reply:
Indeed, thanks!